test/gtest/ucp/test_ucp_tag_match.cc

```diff
@@ -15,6 +15,32 @@ using namespace ucs; /* For vector<char> serialization */
 class test_ucp_tag_match : public test_ucp_tag {
 public:
     using test_ucp_tag::get_ctx_params;
+
+    virtual void init()
+    {
+        test_ucp_tag::init();
+        ucp_test_param param = GetParam();
+        is_req_based_api = param.variant;
+    }
+
+    static std::vector<ucp_test_param> enum_test_params(const ucp_params_t& ctx_params,
+                                                        const std::string& name,
+                                                        const std::string& test_case_name,
+                                                        const std::string& tls)
+    {
+        std::vector<ucp_test_param> tmp, tmp_ret;
+        ucp_test_param test_param;
+
+        tmp = ucp_test::enum_test_params(ctx_params, name, test_case_name, tls);
+        for (std::vector<ucp_test_param>::iterator it = tmp.begin(); it != tmp.end(); it++)
+        {
+            it->variant = false;
+            tmp_ret.push_back(*it);
+            it->variant = true;
+            tmp_ret.push_back(*it);
```

Comment:
No, it does not. It is only applicable to test_ucp_tag_match, as we define this function there